coned/meter.py

```diff
@@ -98,25 +100,37 @@ async def last_read(self):
             # parse the return reads and extract the most recent one
             # (i.e. last not None)
             jsonResponse = json.loads(self.raw_data)
-            lastRead = None
+
+            availableReads = []
             if 'error' in jsonResponse:
+                self._LOGGER.info("got JSON error back = %s", jsonResponse['error']['details'])
                 raise MeterError(jsonResponse['error']['details'])
+            self._LOGGER.info("got JSON response containing reads")
```

Comment:
Can you remove this line.

Reply:
@NickBorgers Sorry, I meant to ask you to remove line 41: `logging.basicConfig(level=logging.INFO)`, not this line.
You can put this line back if you want/need to.